### PR TITLE
feat(build): revert scan output and add matrix branch to title

### PR DESCRIPTION
**Pull request by slonka**
Signed-off-by: slonka <slonka@users.noreply.github.com>

### Checklist prior to review

- [ ] Link to docs PR or issue --
- [ ] Link to UI issue or PR --
- [ ] Is the [issue worked on linked][1]? --
- [ ] The PR does not hardcode values that might break projects that depend on kuma (e.g. "kumahq" as a image registry) --
- [ ] The PR will work for both Linux and Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [ ] Unit Tests --
- [ ] E2E Tests --
- [ ] Manual Universal Tests --
- [ ] Manual Kubernetes Tests --
- [ ] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [ ] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? --
- [ ] Do you need to explicitly set a [`> Changelog:` entry here](../blob/master/CONTRIBUTING.md#submitting-a-patch) or add a `ci/` label to run fewer/more tests?

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

File: .github/workflows/update-insecure-dependencies.yaml [hunk 1]
```diff
@@ -65,9 +67,9 @@ jobs:
             After update:
             ${{ env.SCAN_OUTPUT_AFTER }}
           delete-branch: true
-          title: "chore(deps): security update"
+          title: "chore(deps): security update for ${{ matrix.branch }}"
           draft: false
-          labels: dependencies
+          labels: dependencies ${{ matrix.branch }}
```
Comment (lahabana) [inline .github/workflows/update-insecure-dependencies.yaml:72]:
```suggestion
          labels: dependencies,${{ matrix.branch }}
```

File: .github/workflows/update-insecure-dependencies.yaml [hunk 2]
```diff
@@ -65,9 +67,9 @@ jobs:
             After update:
             ${{ env.SCAN_OUTPUT_AFTER }}
           delete-branch: true
-          title: "chore(deps): security update"
+          title: "chore(deps): security update for ${{ matrix.branch }}"
```
Comment (lahabana) [inline .github/workflows/update-insecure-dependencies.yaml:70]:
```suggestion
          title: "chore(deps): security update
```

The labels are displayed in the PR list so no need to make this redundant

**Review by lahabana — APPROVED** [review]

File: .github/workflows/update-insecure-dependencies.yaml [hunk 3]
```diff
@@ -65,9 +67,9 @@ jobs:
             After update:
             ${{ env.SCAN_OUTPUT_AFTER }}
           delete-branch: true
-          title: "chore(deps): security update"
+          title: "chore(deps): security update
```
Comment (slonka) [inline .github/workflows/update-insecure-dependencies.yaml:70]:
```suggestion
          title: "chore(deps): security update"
```

File: .github/workflows/update-insecure-dependencies.yaml [hunk 4]
```diff
@@ -47,13 +48,14 @@ jobs:
...
+          SCAN_OUTPUT_AFTER=$(osv-scanner --lockfile=go.mod | tr "+" "|" | awk 'NR>3 {print last} {last=$0}' || true)
           echo "SCAN_OUTPUT_AFTER<<EOF" >> $GITHUB_ENV
-          osv-scanner --lockfile=go.mod | tr "+" "|" | awk 'NR>3 {print last} {last=$0}' || true >> $GITHUB_ENV
+          echo "$SCAN_OUTPUT_AFTER" >> $GITHUB_ENV
           echo "EOF" >> $GITHUB_ENV
       - name: "Create Pull Request"
         uses: peter-evans/create-pull-request@v4
         with:
-          commit-message: "chore(deps): security update"
+          commit-message: "chore(deps): security update for ${{ matrix.branch }}"
```
Comment (slonka) [inline .github/workflows/update-insecure-dependencies.yaml:58]:
```suggestion
          commit-message: "chore(deps): security update"
```

**Pull request merged**